query.go

```diff
@@ -638,6 +643,21 @@ checkResponses:
 	}
 }
 
+func (s *ChainService) getFilterFromCache(blockHash *chainhash.Hash,
```

Comment:
Missing `godoc` comments. Also the method could use a bit more "air" for breathability.